rise-financial-es6.js

```diff
@@ -294,7 +294,8 @@
         event: "ready"
       };
 
-      if ( !this._firebaseApp ) {
+      // ensure firebase app has not been initialized already
+      if ( !this._firebaseApp && !firebase.apps.length ) {
```

Comment:
@rodrigopavezi If firebase has allready been initialized and the second test fails then this._firebaseApp isn't set. Shouldn't it be set to the existing instance that caused the second test to fail?